web/autograder_service.go

```diff
@@ -383,13 +383,15 @@ func (s *AutograderService) UpdateGroup(ctx context.Context, in *pb.Group) (*pb.
 
 // DeleteGroup removes group record from the database.
 // Access policy: Teacher of CourseID.
-func (s *AutograderService) DeleteGroup(ctx context.Context, in *pb.RecordRequest) (*pb.Void, error) {
-	usr, err := s.getCurrentUser(ctx)
+func (s *AutograderService) DeleteGroup(ctx context.Context, in *pb.DeleteGroupRequest) (*pb.Void, error) {
+
+	// TODO(vera): will neen an scm passed to the web method in case we want to delete the group repo too (in.WithRepo == true)
```

Comment:
I think we should always delete the team, group repo and the group database entry. Does it make sense to keep either of these, and not all?